tests/robot-tests/tests/admin_and_public/bau/prerelease_visibility.robot

```diff
@@ -27,7 +27,8 @@ Verify release summary
 Upload subject
     [Tags]  HappyPath
     user clicks link  Data and files
-    user waits until h2 is visible  Add data file to release
+    user waits until h2 is visible  Add data file to release  90
```

Comment:
I'm going to remove the 90 wait here too. The new wait below should do the trick.

Adding extra waits should be a tactic of last resort as it lengthens the overall run time, particularly when there are failures.